File: C/Iniciantes/Uri1017_GastoDeCombustivel.c
#include <stdio.h>
#include <math.h>

int main(){
	
	float litros, distancia;
	int tempo, vm;
	
	scanf("%d %d", &tempo, &vm);

	distancia = vm * tempo;
	litros = distancia/12;
	
	printf("%.3f\n", litros);
	
	return 0;
}

File: Javascript/Uri1017_GastoDeCombustivel.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var tempo = parseInt(lines.shift());
var vm = parseInt(lines.shift());

var distancia = vm * tempo;
var litros = distancia / 12;

console.log(litros.toFixed(3));